test/include/dlaf_test/matrix/util_tile.h

```diff
@@ -74,6 +74,30 @@ void print(const Tile<T, Device::CPU>& tile, int precision = 4, std::ostream& ou
   out.precision(out_precision);
 }
 
+/// Create a MemoryView and initialize a Tile on it.
+///
+/// @pre size is the dimension of the tile to be created (type: TileElementSize);
+/// @pre ld is the leading dimension of the tile to be created.
+template <class T>
+Tile<T, Device::CPU> createTile(const TileElementSize& size, const SizeType& ld) {
+  using dlaf::util::size_t::mul;
+  memory::MemoryView<T, Device::CPU> support_mem(mul(ld, size.cols()));
+  return Tile<T, Device::CPU>(size, std::move(support_mem), ld);
+}
```

Comment:
Nitpicks
 
1. Why `createTile()` and not `create_tile()`? I though that we use snake case for free functions (e.g. in `range2d.h`, `util_matrix.h` or in `tile.h`).

2. It's OK to pass the parameters of the function by value, these are small types, no need to have a const ref. Sometimes this helps the compiler with aliasing rules, although, in our case, that's not going to help us much. It's also easier to grok.

Reply:
1. From the coding style:
```
Function Names

Function names should start with a lowercase character and have a capital letter for each new word. Underscores (_) are not allowed.

The only exception to this rule are getter and setter methods for class data members. They follow the naming convention set_variable and get_variable, respectively (see Class Format).
```
Some wrong names might have been missed in reviews.

2. I agree.